theme/layouts/_breadcrumbs.ejs

```diff
@@ -0,0 +1,23 @@
+<%
+const query = site.queryPages;
+if (page.pathname !== '') {
+  const [queryPage] = query(({pathname}) => page.pathname === pathname);
+  const pathnameParts = queryPage.originalPathname.split('/'); %>
+  
+  <nav aria-label='Breadcrumb' id='breadcrumb'>
+  <ol>
+    <% 
+    for (let i=0; i <= pathnameParts.length; i++) {
+      const pickOriginalPathname = pathnameParts.slice(0, i).join("/");
+      const [{pathname, title}] = query(({originalPathname}) => originalPathname == pickOriginalPathname);
+      if (i == pathnameParts.length) { %>
+        <span aria-current='page'><%= title %></span>
```

Comment:
I guess you meant to use `<li>` instead of span.

Reply:
We need a way to address separate CSS for the current page Breadcrumb part. Also we losing `aria-current='page'`.

Reply:
Why are we losing `aria-current='page' `?